sentry-spring/src/main/java/io/sentry/spring/tracing/SentryTracingFilter.java

```diff
@@ -131,7 +131,8 @@ private ITransaction startTransaction(
                 TransactionNameSource.URL,
                 "http.server",
                 new SentryTraceHeader(sentryTraceHeader),
-                baggage);
+                baggage,
+                null);
```

Comment:
shouldn't this be covered by the method overload without spanId?

Reply:
I added another param (`spanId`) to it. The other one doesn't have `baggage` so can't be used here. I didn't want to introduce yet another overload for something that's only used internally anyways. Eventually I'd like to improve this by e.g. introducing a param object that has defaults which can be overriden.